testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -645,6 +647,19 @@ def test_slice_iteration(self, multiverse):
         assert_equal(np.array(frames),
                      np.arange(u.trajectory.n_frames)[4:-2:4],
                      err_msg="slicing did not produce the expected frames")
+        assert_equal(u.trajectory.ts.frame, 0,
+                     err_msg="slice iterator did not rewind")
```

Comment:
Same as previous comment: Is this test actually testing your code? Did it fail *before* you added your fix? If not, remove the test.